articles/client-platforms/jquery/11-api-authorization.md

```diff
@@ -0,0 +1,32 @@
+---
+title: OAuth2 API Authorization
+description: This tutorial demonstrates how to use Auth0 to make authorized API calls from your web app.
+---
+
+<%= include('../../api-auth/_preview-warning') %>
+<%= include('./_includes/_compat_warning') %>
+
+### Before Starting
```

Comment:
Sure, something like the first part of this doc? https://auth0.com/docs/api-auth

We could also provide a link to that for more information.